modin/pandas/test/test_io.py

```diff
@@ -212,6 +212,44 @@ def eval_to_file(tmp_dir, modin_obj, pandas_obj, fn, extension, **fn_kwargs):
     assert assert_files_eq(unique_filename_modin, unique_filename_pandas)
 
 
+def eval_to_csv_file(tmp_dir, modin_obj, pandas_obj, extension, **kwargs):
+    if extension is None:
+        kwargs["mode"] = "t"
+        kwargs["compression"] = "infer"
+        modin_csv = modin_obj.to_csv(**kwargs)
+        pandas_csv = pandas_obj.to_csv(**kwargs)
+        if modin_csv == pandas_csv:
+            return
```

Comment:
when this condition is false?

Reply:
The text representation could be different, for example, dates format - `2023-01-01 00:00:01.000000000` vs `2023-01-01 00:00:01`. However, the values are still valid and, if loaded back to pandas, we will get an identical data frame.